modules/ipi-install-configuring-host-network-interfaces-for-subnets.adoc

```diff
@@ -12,7 +12,9 @@ If you have established different network segments or subnets for remote worker
 
 [IMPORTANT]
 ====
-All control plane nodes must run in the same subnet. When using more than one subnet, you can also configure the Ingress VIP to run on the control plane nodes by using a manifest. See "Configuring network components to run on the control plane" for details.
+You can run control plane nodes in the same subnet or multiple subnets. A multiple subnet environment is useful for reducing the risk of your {product-title} cluster from failing because of a hardware failure or a network outage.
+
+For a multiple subnet environment, create a manifest configuration file and specify the external load balancer address in the `ingressVIPs` and `apiVIPs` parameters of the file. Ensure that you configure an external load balancer in place of the default load balancer by specifying `UserManaged` in the `loadBalancer.type` parameter of the `install-config.yaml` file. For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".
```

Comment:
Nit/suggestion to make that second sentence a little shorter.
```suggestion
For a multiple subnet environment, create a manifest configuration file and specify the external load balancer address in the `ingressVIPs` and `apiVIPs` parameters of the file. Configure an external load balancer instead of the default load balancer by specifying `UserManaged` in the `loadBalancer.type` parameter of the `install-config.yaml` file. For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".
```